### DISCUSSION TITLE
dials.find_spots should not save shoeboxes

### DISCUSSION THREAD

**Discussion by graeme-winter** [dials/dials#2412] [Ideas]
They are big and expensive and I don't think we need them

Also: anywhere in the code do we use 

```
| xyzobs.px.variance     | 0.0833, 0.0833, 0.0833 | 0.2722, 0.2660, 0.2500    | 0.0876, 0.0887, 0.0838   |
```

and

```
| intensity.sum.variance | 1.0                    | 91780.0                   | 119.9   
```

which as far as I can tell, in the latter case, takes the NULL view of being the same as the value.

**Comment by graeme-winter** [comment]
This is presented as an _opinion_. I would welcome some challenges / other opinions

**Comment by phyy-nx** [comment]
I believe diffBragg uses shoeboxes, but I don't know that it uses the strong shoeboxes.  It may go back to the original image and remake larger shoeboxes.  The strong shoeboxes are pretty tight around the foreground pixels.

I can ask around.

intensity.sum.variance is used quite a lot but I don't know that xyzobs.px.variance is.

**Comment by nksauter** [comment]
We use spotfinder shoeboxes from dials.stills_process in our diffBragg
analysis.  Fine if it is not default, but we need the option.
Nick
